extensions.md

```diff
@@ -122,49 +136,69 @@ further information (like definition location, which may require consulting an
 
 ## UTF-8 offsets
 {:.v9}
 
-LSP specifies that offsets within lines are in UTF-16 code units (for `Position`s and also delta-encoded document updates).
-This unusual is an unfortunate legacy of VSCode's JavaScript implementation.
+LSP specifies that offsets within lines are in UTF-16 code units (for
+`Position`s and also delta-encoded document updates). This unusual is an
+unfortunate legacy of VSCode's JavaScript implementation.
 
-clangd allows clients to use UTF-8 offsets instead. This allows clients that always speak UTF-8 (in violation of the protocol) to work correctly, and those that are UTF-8 native to avoid unneccesary transcoding (which may be slow if implemented in e.g. vimscript).
+clangd allows clients to use UTF-8 offsets instead. This allows clients that
+always speak UTF-8 (in violation of the protocol) to work correctly, and
+those that are UTF-8 native to avoid unnecessary transcoding (which may be
+slow if implemented in e.g. VimScript).
 
 **New client capability**: `offsetEncoding : string[]`:
 
-  Lists the encodings the client supports, in preference order. It SHOULD include `"utf-16"`. If not present, it is assumed to be `["utf-16"]`
-  
-  Well-known encodings are:
-  - `utf-8`: `character` counts bytes
-  - `utf-16`: `character` counts code units
-  - `utf-32`: `character` counts codepoints
+Lists the encodings the client supports, in preference order. It SHOULD
+include `"utf-16"`. If not present, it is assumed to be `["utf-16"]`
+
+Well-known encodings are:
+
+- `utf-8`: `character` counts bytes
+- `utf-16`: `character` counts code units
+- `utf-32`: `character` counts codepoints
 
 **New InitializeResponse property**: `offsetEncoding: string`:
-  - Specifies the encoding that was selected by the server, and should be used.
-  - This should be one of the requested encodings, or `"utf-16"` if none ore supported.
-  - Only sent if the client capability was specified (or the equivalent command-line flag `-offset-encoding=utf-8`).
-  
+
+- Specifies the encoding that was selected by the server, and should be used.
+- This should be one of the requested encodings, or `"utf-16"` if none ore
+  supported.
+- Only sent if the client capability was specified (or the equivalent
+command-line flag `-offset-encoding=utf-8`).
+
 **Advice for clients using this extension**:
-  - clients that only support UTF-8 should send `offsetEncoding: ["utf-8"]` in their client capabilities.
-    This will cause the server to use UTF-8 if supported.
-  - clients that prefer UTF-8 but can use UTF-16 should send `offsetEncoding: ["utf-8", "utf-16"]` and observe the selected encoding in the `InitializeResponse`, defaulting to UTF-16 if it's not present.
-    This will negotiate UTF-8 with servers that support it.
-  - clients that prefer UTF-16 may send `offsetEncoding: ["utf-16"]` or simply not use the extension.
- 
+
+- clients that only support UTF-8 should send `offsetEncoding: ["utf-8"]` in
```

Comment:
these seem to have been rewrapped quite short

(FWIW, when editing markdown, I'm usually in an editor with softwrap on, and typically 90-120 cols, so I find too-short lines more of a problem than too-long ones. I'm sure workflows vary though!)